Automattic/jetpack
# AI Assistant: Display AI sidebar on first highlight hover

Fixes #

## Proposed changes:

* Renames the localStorage entries
* Displays the AI Assistant section of the Jetpack sidebar on *first* hover

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the editor, close the AI Assistant section and the sidebar. Since this only happens on first load, be careful not to hover any highlight in doing so
* Hover any highlight
* Check that the AI Assistant section is displayed
* Close the AI Assistant section
* Check that hovering a highlight again does not display de sidebar again
